library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -77,6 +77,14 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
   void SetRaytracingSamples(int samples);
   ///@}
 
+  ///@{
+  /**
+   * Set/Get SetUseOrthographicProjection
+   */
+  void SetUseOrthographicProjection(bool use);
+  vtkGetMacro(UseOrthographicProjection, bool);
```

Comment:
if you dont need the getter, do not declare it.

Reply:
Will remove, thanks